bpf/lib/egress_gateway.h

```diff
@@ -1,12 +1,22 @@
 /* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2021 Authors of Cilium */
 
-#ifndef __LIB_EGRESS_POLICIES_H_
-#define __LIB_EGRESS_POLICIES_H_
```

Comment:
Can we keep this and the filename as is? It was done on purpose since this same file will inherit the SRv6 egress policies as well.